srml/support/src/traits.rs

```diff
@@ -65,6 +65,49 @@ pub trait ArithmeticType {
 	type Type: SimpleArithmetic + As<usize> + As<u64> + Codec + Copy + MaybeSerializeDebug + Default;
 }
 
+/// Simple payment making trait, operating on a single generic `AccountId` type.
+pub trait MakePayment<AccountId> {
+	/// Make some sort of payment concerning `who` for an extrinsic (transaction) of encoded length
```

Comment:
can we mention this is designed for transaction payment instead of "some sort of payment"? unless this can be used for other purpose?